execute_pipeline.py

```diff
@@ -104,8 +143,19 @@ def _parse_args():
   parser = _CreateArgumentParser()
   flags = parser.parse_args()
 
+  repo_root = flags.reporoot
   pipeline_args = {}
-  repl_vars = {"REPOROOT": flags.reporoot}
+
+  if  flags.env.lower() != 'local':
+    pipeline_id = str(uuid.uuid4())
+    # For remote execution, repo_root will be a random temp directory.
+    repo_root = os.path.join('/', 'tmp', 'artman', pipeline_id)
```

Comment:
Better to use mkdtemp: https://docs.python.org/2/library/tempfile.html#tempfile.mkdtemp